wavefront/values.yaml

```diff
@@ -80,8 +80,28 @@ collector:
   #   env: production
   #   region: us-west-2
 
+  ## If set to true, host OS metrics will be collected
+  hostOSMetrics: false
+
   ## Filters to apply towards all metrics collected by the collector
   filters:
+    # Optimized metrics collection to focus collection on default dashboard support.
```

Comment:
Maybe reword this to:

`# Optimized metrics collection to omit peripheral metrics.`

Reply:
Done